bin/ember-template-lint.js

```diff
@@ -258,7 +258,14 @@ async function run() {
   let options = parseArgv(process.argv.slice(2));
   let positional = options._;
   let config;
-  let todoInfo;
+  let todoInfo = {
+    added: 0,
+    removed: 0,
+    todoConfig: options.updateTodo
+      ? getTodoConfig(options.workingDirectory, getTodoConfigFromCommandLineOptions(options))
+      : {},
+  };
```

Comment:
It's _probably_ always fine to include the todoConfig

Reply:
Is this an actionable comment?

Reply:
I assume you mean

```suggestion
    todoConfig: getTodoConfig(options.workingDirectory, getTodoConfigFromCommandLineOptions(options)),
  };
```

Reply:
ya exactly

Reply:
Agree and done.